tests/Platforms/OraclePlatformTest.php

```diff
@@ -617,7 +618,7 @@ public static function dataCreateSequenceWithCache(): iterable
      */
     protected function getAlterTableRenameIndexSQL(): array
     {
-        return ['ALTER INDEX idx_foo RENAME TO idx_bar'];
+        return ['ALTER INDEX "idx_foo" RENAME TO idx_bar'];
```

Comment:
This is a wrong change. The column isn't quoted in the test and shouldn't be quoted in the resulting statement.

Reply:
The index name is in lower case, so it must have been quoted initially. Otherwise, it would be IDX_FOO and thus, the test is correct

Reply:
The index name contains lower case characters so the name must have been quoted initially. Otherwise, the index name returned by Oracle would be `IDX_FOO`. Thus, the change is correct.

Reply:
The index name is defined here in a portable lower-cased way, and it's not quoted:
https://github.com/doctrine/dbal/blob/5dd30ad24dde117ca0f272d370dbafe30fc2a5a1/tests/Platforms/AbstractPlatformTestCase.php#L878-L880

It would have to be quoted if it was introspected from the database schema:
https://github.com/doctrine/dbal/blob/4afa4917d6bf3898ef62d49accc708a573b21c6b/src/Schema/OracleSchemaManager.php#L312-L316